Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -2285,20 +2411,28 @@ - (void)searchBarCancelButtonClicked:(UISearchBar *)searchBar {
 //    }];
 //    coordinatorBridgePresenter = nil;
 //}
-
+//
 //- (void)createRoomCoordinatorBridgePresenterDelegateDidCancel:(CreateRoomCoordinatorBridgePresenter *)coordinatorBridgePresenter
 //{
 //    [coordinatorBridgePresenter dismissWithAnimated:YES completion:nil];
 //    coordinatorBridgePresenter = nil;
 //}
+//
+//- (void)createRoomCoordinatorBridgePresenterDelegate:(CreateRoomCoordinatorBridgePresenter *)coordinatorBridgePresenter didAddRoomsWithIds:(NSArray<NSString *> *)roomIds
+//{
+//    [coordinatorBridgePresenter dismissWithAnimated:YES completion:nil];
+//    coordinatorBridgePresenter = nil;
+//}
 
 #pragma mark - Empty view management
 
 - (void)showEmptyViewIfNeeded
 {
+    // Tchap: Not available in Tchap
```

Comment:
@phlniji don't you think we should support this empty view option in Tchap now we are based on Element-iOS code.
I think this view is displayed mainly for new users with empty rooms list. It may suggest actions like room creation. If this is the case, can you please create a new issue in Tchap-iOS to restore this empty view (sharing the current Element-iOS empty view display in this issue will help to define the plan)

Reply:
As requested before, I disabled this view in a previous rebase.
But I think it's useful to have it.
So I will add an issue for this.